worlds/tits_the_3rd/client/client.py

```diff
@@ -63,6 +64,9 @@ def install_game_mod(self):
         scena_temp_folder = lb_ark_folder / "ED6_DT21_BASE"
         game_mod_folder = lb_ark_folder / "ED6_DT21"
         os.makedirs(lb_ark_folder, exist_ok=True)
+        if "player.txt" in os.listdir(lb_ark_folder):  # TODO: Check for player id + seed
+            return True
```

Comment:
Why do we return here?

Reply:
This is purely for dev so the client doesn't override our scena change every time. The TODO is to properly read the file and check if the player has not changed and therefore skip the patching process